packages/squiggle-lang/src/rescript/symbolicDist/SymbolicDistTypes.res

```diff
@@ -31,6 +31,7 @@ type triangular = {
   high: float,
 }
 
+@genType
```

Comment:
I feel kind of awkward about this. Our internal structure should change a fair amount in the next few weeks/months. I don't want to have a big typescript interface we need to commit to.

It's fine if this is just for a few things now, and you'll take responsibility of it. At least, we shouldn't document the parts that need this. (I really don't want to commit to keeping such an API)

Reply:
I attempted to get rescript to generate opaque types for some things, to no avail... It might be possible with some more experimentation? But not really sure how I would go about fixing that